mythril/analysis/report.py

```diff
@@ -231,7 +231,11 @@ def as_swc_standard_format(self):
                     },
                     "severity": issue.severity,
                     "locations": [{"sourceMap": "%d:1:%d" % (issue.address, idx)}],
-                    "extra": {},
+                    "extra": {
+                        "discoveryTime": "{}".format(
```

Comment:
This is the wrong type right? A string, while it should be an integer

Reply:
Changed :+1: